src/locales/it/translation.json

```diff
@@ -233,7 +233,7 @@
                 "5": "Set up Monsters:"
             },
             "monsterFightDeck": "Shuffle the Monster Fight deck and place it (face down) near the game board.",
-            "startingPlayer": "Determine the starting player randomly, or it is whoever most recently read a Witcher book. Players choose Player Boards via any method they wish; see p. 7 of core rules for a fair drafting method.",
+            "startingGiocatore": "Determine the starting player randomly, or it is whoever most recently read a Witcher book. Giocatores choose Giocatore Boards via any method they wish; see p. 7 of core rules for a fair drafting method.",
```

Comment:
Any values on the left should be left as they are. These are the keys that will be found in code and replaced with the value associated with it, dependent upon the currently active language. Apologies for not providing guidance on this when you started!

Reply:
My fault, I'm aware of the key/value pairs use but a first attempt was a brutal "Replace All" of the word through VSCode, I quickly realized was not the right way and I reverted, but seems I've not reverted all. 😅
Next commit will fix that, thanks for putting an eye on 💪